core/src/trezor/lvglui/scrs/template.py

```
@@ -4952,3 +4972,141 @@
                     cancel_text=None,
                     page_size=351,
                 )
+
+
+class Turbo(FullSizeWindow):
+    def __init__(
+        self,
+        message_text=_(i18n_keys.MSG__UNKNOWN_MESSAGE),
```

Ruff (0.11.9)

Do not perform function call `_` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)